src/wallet/wallet2.cpp

```diff
@@ -3476,7 +3475,6 @@ void wallet2::refresh(bool trusted_daemon, uint64_t start_height, uint64_t & blo
       if (!first && blocks.empty())
       {
         m_node_rpc_proxy.set_height(m_blockchain.size());
-        refreshed = true;
```

Comment:
I think this was set by the author to facilitate easier debugging. 

I personally don't need it, but just saying that's the reason I haven't removed it myself.

Reply:
That was my thought as well. I don't see any need to remove it.

Reply:
@hyc pushed changes to keep the flag in and remove the warning

Reply:
@jeffro256 "Easier debugging" was speculation. As it says in mooo's comment, he wanted to clean it up but probably forgot about it. We can ask him again to be sure but just silencing doesn't make much sense here.

Reply:
Pinging @moneromooo-monero

Reply:
```
12:34 <moneromooo> I had to reorder some code to fix... a timing info leak IIRC. In turn, this undid something I had fixed before, which I believe was a subtle race condition with the txpool.
12:35 <moneromooo> It was pretty subtle IIRC, and so I needed time to think about how to refix it after the move, and I never got to it. I do not remember the details of the race.
12:35 <moneromooo> I don't think I'm likely to ever get to it now.
```

So either we remove the unused variable or we keep it. Silencing doesn't make sense in my opinion.

Reply:
+1 for commenting out rather than silencing.

There will surely be a lot of debugging to do there in future, but then it can be uncommented on demand, still not causing warnings to everybody else and not having to resort to compiler hacks, which might not be cross-platform.

Reply:
Probably should add some of mooo's commentary as a comment in the source too. Saves digging thru github issues to find out what it's all about.

Reply:
done